### ISSUE TITLE
Detach method not returning a pure Pojo [moved]

### ISSUE THREAD

**Issue by lvca** [orientechnologies/orientdb#988]
This is [Issue 988](http://code.google.com/p/orient/issues/detail?id=988) moved from a Google Code project.
Added by 2012-07-30T19:41:16.000Z by [ajitem...@gmail.com](http://code.google.com/u/118401179941031299021/).
Please review that bug for more context and additional comments, but update this bug.
 Closed (Fixed).

Original labels: Type-Defect, Priority-Medium, v1.2
### Original description

```
OrientDB release? (If you're using a SNAPSHOT please attach also the build number found in &quot;build.number&quot; file): orientdb-1.1.0-20120719.010924


<b>What steps will reproduce the problem?</b>
1. Use the OObjectDatabaseTx API to create a database.
2. Create an object in a database file.
3. Read the object back from the database.
4. Use the db.detach() method to extract the Pojo.
5. The object returned is not a pure Pojo of the original object.

<b>If you're using custom settings please provide them below (to dump all the</b>
<b>settings run the application using the JVM argument</b>
-Denvironment.dumpCfgAtStartup=true): No custom settings.


What is the expected output? What do you see instead?: Expected to return a pure Pojo. Instead the object looks like a proxied object. I was using GWT-RPC in my test and when sending the object over the wire as a GWT-RPC response, I got a serialization error since it included some artifacts from the proxied object that were not serializable.


<b>If you're describing a performance or memory problem the profiler dump can</b>
<b>be very useful (to dump it run the application using the JVM arguments</b>
<b>-Dprofiler.autoDump.reset=true -Dprofiler.autoDump.interval=10</b>
-Dprofiler.enabled=true): Not applicable


<b>Please use labels and text to provide additional information.</b>
```
